uint8 BOUNDING_BOX=1
uint8 ENV_CONSTRAINT=2

uint8 type
uint32 box_id
string constraint_name

bool terminate_on_collision
